src/server/game/Movement/MotionMaster.h

```diff
@@ -38,21 +40,30 @@ namespace Movement
     class MoveSplineInit;
 }
 
+class MovementSlot
+{
+    public:
+        MovementSlot(uint8 type);
+        ~MovementSlot();
```

Comment:
Should `MovementSlot` be renamed to `MovementSlots` here too (+ `class` , `uint8 type`) ?
build error: https://travis-ci.org/TrinityCore/TrinityCore/builds/371823289#L1385-L1391

Reply:
dont worry about build errors, yet